userprofile/templates/userprofile/login.html

```diff
@@ -20,6 +20,29 @@ <h1>{% blocktrans %}Log in to {{ brand_name }}{% endblocktrans %}</h1>
 	<div id="login-box-row" class="row">
 
 		{% if shibboleth_login %}
+			<section aria-labelledby="login-shibboleth-users" class="col-sm-6 col-md-3 login-box">
```

Comment:
`aria-labelledby` values and the target element ids do not match. Check both Shibboleth and Haka sections in this template. Each section needs a unique aria label. In this case, the label is a normal visible `<h2>` title and `aria-labelledby` refers to its id attribute.

So, this one should stay as `login-shibboleth-users`, but the Haka section below needs to be corrected, right?